ShdrPlygrnd/materials/masonJarWater.mtlx

```diff
@@ -10,15 +10,13 @@
   </image>
 
   <open_pbr_surface name="mtlxopen_pbr_surface" type="surfaceshader">
-    <!-- FIXME: Coat is not enabled by default -->
-    <input name="coat_ior" type="float" value="1" />
-    <input name="coat_roughness" type="float" value="0.1" />
     <input name="specular_weight" type="float" value="0.5" />
     <input name="specular_ior" type="float" value="1.52" />
     <input name="specular_roughness" type="float" value="0.1" />
     <input name="specular_roughness_anisotropy" type="float" value="0.5" />
+    <input name="subsurface_weight" type="float" value="1" />
     <input name="subsurface_color" type="color3" nodename="ScatterColor" />
-    <input name="subsurface_radius" type="float" value="5" />
+    <input name="subsurface_radius" type="float" value="0.5" />
```

Comment:
Why ? Does this make a meaningful appearance change ?

Reply:
Thanks for those extra tests !
It is the main intent is to get some murky water which you have correctly addressed.
About Arnold / Octane differences, I would not spend more time on this for now.